packages/cli/src/Db.ts

```diff
@@ -45,6 +45,8 @@ export function linkRepository<Entity>(entityClass: EntityTarget<Entity>): Repos
 export async function init(
 	testConnectionOptions?: ConnectionOptions,
 ): Promise<IDatabaseCollections> {
+	if (isInitialized) return collections;
```

Comment:
During test, prevent command from reinitializing `Db` and so from overwriting test DB connection options with actual DB.